views/includes/comments.tx

```diff
@@ -168,10 +169,15 @@
 
 <a name='comments'></a>
 
+    <h3>This blog has been archived</h3>
+    <p>
+    Thank you for reading and contributing lively discussion to our blog!
+    Please feel free to read new posts from DuckDuckGo on <a href="https://spreadprivacy.com/">spreadprivacy.com</a>
```

Comment:
Looks good. How about a little more appeal, for example:
`Read more posts about online privacy on our new blog at <a href="https://spreadprivacy.com/">spreadprivacy.com</a>.`